components/ModalTemplateAddGridWidget.vue

```diff
@@ -38,12 +38,12 @@ const selectedProperties = computed(() => {
 
 const handleSave = () => {
   if (selectedWidgetType.value) {
-    const newWidget: GridWidget = {
+    const newWidget: GridWidgetWithoutCords = {
       type: GRID_WIDGET_TYPE[selectedWidgetType.value.value as GridWidgetType],
-      width: 1,
-      height: 1,
+      w: 1,
+      h: 1,
       properties: { ...formValues.value },
-      id: uuidv4(),
+      i: uuidv4(),
```

Comment:
To be honest `i` does not associate with `id` but rather with a counter from a loop 🙂
What is the idea driving these renames? 🤔

Reply:
Thanks, actually I wanted to change that so `i` is not index of elements. Initially I thought that `i` suppose to be index but based on the library spec it's id of the element, so can be anything. That's why I removed previously added `id` field as redundant and changed that `i` is generated by `uuid` library.

Reply:
We could call it "id" one additional character doesn't really matter much.

Reply:
Grid library already provides us `i` so if we add `id` like before we will have two identifiers which imo is not necessary.